ejb3/src/main/java/org/jboss/as/ejb3/timerservice/schedule/CalendarBasedTimeout.java

```diff
@@ -539,47 +530,6 @@ private boolean isDayOfMonthWildcard() {
         return this.scheduleExpression.getDayOfMonth().equals("*");
     }
 
-    private void nullCheckScheduleAttributes(ScheduleExpression schedule) {
```

Comment:
@chengfang Are we certain that those are not null? Are all those parts of code reduntant?

Reply:
These 7 fields all have non-null defaults: https://github.com/jakartaee/enterprise-beans/blob/master/api/src/main/java/jakarta/ejb/ScheduleExpression.java#L533-L540

So unless the user intentionally set them to null, they will not be null.

Reply:
ok, thanks for claryfing